massgov/mayflower
# DP-19332 The nav bar positioning

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Position the nav bar below the site wide alert container regardless of the screen width.

Add the alert to the hamburger menu sample page:  http://localhost:3000/patterns/05-pages-Homepage-hamburger/05-pages-Homepage-hamburger.html

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-19332)


## Steps to Test


Note: 

- This needs to be tested in Drupal.  The issue is not seen in Mayflower.  <-- UPDATE:  You can test it in Mayflower since the alert component is added to the hamburger menu sample page:  http://localhost:3000/patterns/05-pages-Homepage-hamburger/05-pages-Homepage-hamburger.html
- The overlay is still over the nav bar as the hamburger menu is open.  This will be handled in a separate ticket.

1. Build the site with the Drupal branch `DP-19120-global-nav-drupal-changes` with this MF branch `patternlab/DP-19332_nav-bar-positioning`.
2. Check no visual and functional change in the navigation bar and the hamburger menu in screen width over 840px.
3. With the screen width 840px and below, check the nav bar stays below the site wide alert in any screen sizes.
4. In the same screen width setting, as you scroll the page, check the nav bar moves along with the other elements in the page.
5. In the same screen width setting, check the hamburger menu function doesn't change from the current implementation. 


## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
